### PR TITLE
CB-8634 `cordova platform add git_url` should accept custom branches and check them out

### PR DESCRIPTION
**Pull request by omefire**
CB-8634 `cordova platform add git_url` should accept custom branches and check them out

### CODE REVIEW

**Comment by omefire** [comment]
Related PR: https://github.com/apache/cordova-lib/pull/180

**Comment by omefire** [comment]
@vladimir-kotikov

File: doc/platform.txt [hunk 1]
```diff
@@ -29,11 +29,14 @@ Manage project platforms
 
 Syntax
     <plat-spec>:
-        <platform-ver>|<path>|<url>
+        <platform-ver>|<path>|<url-branch>
```
Comment (vladimir-kotikov) [inline doc/platform.txt]: :nit: IMHO it's better to use `<gti-url>` instead of `<url-branch>`

File: doc/platform.txt [hunk 2]
```diff
@@ -42,6 +45,8 @@ Syntax
     <url> ............................. url to a git repository containing a platform
 
     <version> ......................... major.minor.patch version specifier
+
+    <branch> .......................... name of the branch to checkout after cloning the git repo
```
Comment (vladimir-kotikov) [inline doc/platform.txt]: If `#branch` is not specified after repo url, then `master` will be used by default, right? Could you please note this as well.

**Comment by vladimir-kotikov** [comment]
LGTM

File: doc/platform.txt (hunk 2 above)
Comment (omefire) [inline doc/platform.txt]: Thanks! done.

**Pull request closed without merging**